### PR TITLE
Base app

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
- [ ] Config for audio/ussd 
- [x] Config for audio file names
- [x] Choice state
- [x] Free text state 
- [x] End State

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

**Comment by imsickofmaps** [comment]
👍

File: test/app.test.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                    name: 'connectivity-test',
+                    testing_today: '2016-05-31 06:07:08.999',
+                    voice_enabled: true,  // default
+                    services: {
+                        voice_content: {
+                            api_token: 'test_token_voice',
+                            url: "http://localhost:8001/api/v1/"
+                        }
+                    },
+                    state_prompts: {
```
Comment (imsickofmaps) [inline test/app.test.js:22]: let's do `state_audio` too, and link to full audio file

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps good to land?

**Comment by imsickofmaps** [comment]
👍 🚢

**Pull request merged**